data/json/npcs/robofac/robofac_intercom/robofac_intercom_missions.json

```diff
@@ -1,4 +1,156 @@
 [
+  {
+    "//1": "This is the main talk topic for all of the intercom's contracts.  The player is able to learn more about missions here, and accept new ones.",
+    "//2": "This also contains dialogue topics to complete one-time missions.  Repeatables are handled in 'robofac_intercom_trades.json' instead.",
+    "id": [
+      "TALK_ROBOFAC_INTERCOM_CONTRACTS",
+      "MISSION_ROBOFAC_INTERCOM_2_ACCEPTED",
+      "MISSION_ROBOFAC_INTERCOM_ROBOT_SM_1_ACCEPTED",
+      "MISSION_ROBOFAC_INTERCOM_PORTAL_SM_1_ACCEPTED"
+    ],
+    "type": "talk_topic",
+    "dynamic_line": "What would you like to discuss?",
+    "responses": [
+      {
+        "text": "I'm back from the convoy.  [Hand over the hard case.]",
+        "condition": { "and": [ { "u_has_mission": "MISSION_ROBOFAC_INTERCOM_2" }, { "u_has_item": "robofac_hardcase" } ] },
+        "effect": [
+          { "u_sell_item": "robofac_hardcase", "count": 1 },
```

Comment:
should probably be u_consume_item so that you can't randomly buy it back

Reply:
Yeah, I was thinking about that too. `consume_item` doesn't give a popup that you give the item to the NPC, though, so I'm not sure whether or not it would be fine as-is or if it could benefit from some kind of new addition or argument to show the popup. What do you think?

Reply:
I think consume_item needs a popup, but that's outside the scope of this IMO and would be a great QOL in general.

Reply:
Kay. I'll replace it with `u_consume_item` for the time being and then in a later PR I can add an optional argument to show a popup for selling the item.

Reply:
If you add "TRADER_AVOID" to the case it shouldnt be buyable. I think